dashboard/app/views/pd/workshop_mailer/teacher_follow_up.html.haml

```diff
@@ -44,9 +44,29 @@
       style: 'width: 180px; height: 36px; border: 0; color: white; background-color: #FDC62C; font: bolder 18px Arial; display: block; text-align: center; line-height: 36px; text-decoration: none; border-radius: 5px;'}
       Get started here
 
+  - has_partner = !!(@regional_partner && @regional_partner.contact_email)
+  - has_facilitator = @facilitators.length > 0
+  - after_teacher_contact = '.'
+  - if has_facilitator || has_partner
+    - after_teacher_contact = has_facilitator && has_partner ? ',' : ' or'
+
   %p
-  Remember that you can always reach out to us for support at
-  = mail_to('teacher@code.org') + '.'
+    Remember that you can always reach out to us for support at
```

Comment:
Should we move this logic out of haml and into a helper function in workshop mailer? It would be easier to reason and easier to test.

Reply:
Seems like a good idea! Did get me to learn about `succeed` in HAML though :).

Reply:
Probably a good idea :) I looked briefly at doing this and couldn't find a good way to embed the email tags, but can look more. Do you know a good way to send html over to a haml file ? (we need to have links around the emails)

Reply:
moved this logic to workshop mailer